fmriprep/workflows/bold/resampling.py

```diff
@@ -620,7 +628,8 @@ def init_bold_preproc_trans_wf(
                                      (("bold_file", _first), "reference_image")]),
         (inputnode, merge, [("name_source", "header_source")]),
         (merge_xforms, bold_transform, [("out", "transforms")]),
-        (bold_transform, merge, [("out_files", "in_files")]),
+        (bold_transform, threshold, [("out_files", "in_file")]),
```

Comment:
This would actually threshold the "preprocessed" echos, right? (which is actually the fix to the problem, as the boldref is not the target here).

Reply:
Yes, this thresholds the preprocessed echos (or the preprocessed BOLD, if single echo).